src/components/comments.js

```diff
@@ -49,9 +75,15 @@ return(
     )
 })}
 </div>
+<form className="postComment" onSubmit={handleSubmit} >
```

Comment:
This is minor but you could probably extract out this functionality into its own "CommentAdder" component